sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -162,10 +168,7 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
         <configuration>
-          <systemPropertyVariables>
-            <!-- Parallel runs disabled due to concurrency issues when registering the global tracer. -->
-            <junit.jupiter.execution.parallel.enabled>false</junit.jupiter.execution.parallel.enabled>
-          </systemPropertyVariables>
+          <parallel>all</parallel>
```

Comment:
This configuration actually doesn't do anything when using JUnit 5, so the whole plugin could be removed